rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2023-03-21"
```

Comment:
When we decide to change toolchain, we must update this line and [this line](https://github.com/sora-xor/sora2-network/blob/master/housekeeping/docker/env/Dockerfile#L47).
Probably we want to do it now for #709

Reply:
We have the same version of Rust in docker and we can't update it before Substrate update because of problems with wasm